examples/tictactoe/src/util/util.ts

```diff
@@ -1,6 +1,6 @@
-import { Player } from "../types";
+import { type PlayField, type Player } from "../types";
 
-export const calculateWinner = (squares: Array<Player>):Player | null => {
+export const calculateWinner = (squares: PlayField[]): Player | undefined => {
```

Comment:
You need to also update the return value `let result: Player | undefined = undefined;`